cernopendata/templates/cernopendata_theme/page.html

```diff
@@ -52,5 +52,22 @@
         });
     </script>
   {% endblock %}
-{% endblock %}
 
+  {%- block mathjax_support %}
+    <script type="text/x-mathjax-config">
+      MathJax.Hub.Config({
+        extensions: ["tex2jax.js"],
+        jax: ["input/TeX", "output/HTML-CSS"],
+        tex2jax: {
+          inlineMath: [ ['$','$'], ["\\(","\\)"] ],
+          displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
+          processEscapes: true
+        },
+        "HTML-CSS": { availableFonts: ["TeX"] }
+      });
+    </script>
+
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS_HTML"></script>
```

Comment:
Keep in mind this https://www.mathjax.org/cdn-shutting-down/ .

Reply:
Hmm, I've seen that I thought that I had taken it into account. I will check it once again.

Reply:
Did I misunderstood something?
The PR uses the recommended _cdnjs.cloudflare.com_ URL

> The easiest way for most site owners will be to simply switch to another CDN provider.
> 
> We recommend cdnjs which also uses CloudFlare for delivery (and on the higher “enterprise” level!). We have been in touch with cdnjs’s maintainers and will help push future MathJax releases to cdnjs.
> 
> For example, if you have been using the latest MathJax version (v2.7.0) change
> 
> <script type="text/javascript" async
>   src="https://cdn.mathjax.org/mathjax/2.7-latest/MathJax.js?...">
> </script>
> 
> to
> 
> <script type="text/javascript" async
>   src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?...">
> </script>
>

Reply:
The comment was mostly for considering adding cdn as a bundle via npm and completely get rid of the cdn. Check this https://www.npmjs.com/package/mathjax . But it is ok for now we can leave it as it is. Just to keep in mind for future.

Reply:
OK. Thanks.